src/components/side_panel/chart/line_bar_pie_panel/config_panel.ts

```diff
@@ -26,6 +38,10 @@ export class LineBarPieConfigPanel extends Component<Props, SpreadsheetChildEnv>
   private state: ChartPanelState = useState({
     datasetDispatchResult: undefined,
     labelsDispatchResult: undefined,
+    headerPosition: {
+      direction: "ROW",
```

Comment:
I don't think we need to support "column".
It's enough for most cases.
Keep it simple :)